11-Backbonejs/01-Introduction-to-Backbonejs.md

```diff
@@ -1,67 +1,60 @@
 
 ## Backbone MV*
 
-Backbone implements the familiar MVC architecture from Rails in a slightly different way.  In Backbone the views & Controllers are combined, thus Rails is called an MV* architecture.  
+Backbone implements the familiar MVC architecture that we learned in Rails in a slightly different way. In Backbone the views & controllers are combined, thus Backbone is called an MV* architecture.  
 
 ### Backbone has 5 Components
 
-1.  Events:  Events are a module that can be mixed in to any object, giving the object the ability to bind and trigger custom named events.  
-1.  Models:  These include the application data and business logic.  These are similar to Rails models.
+1.  **Events**:  Events are a module that can be mixed in to any object, giving the object the ability to bind and trigger custom named events.  
+1.  **Models**:  These include the application data and business logic.  These are similar to Rails models.
 	- Models:
 		- Organize data and business logic.  
 		- Load and save data from and to the server.   
 		- Emit events when data changes.  
-1.  Collections:  These are ordered sets of models
-1.  Views:  These render the models and listen for DOM/Model events.  Views can be used to render a single model to render or a collection.  
-1.  Routers:  Used for Single Page Applications, routers let you add paths to the URL so that you can maintain history and allow users to jump to a specific state in your single-page-application.  A router always contains at least one route and a function to map that particular route to.  
+1.  **Collections**:  These are ordered sets of models
+1.  **Views**:  These render the models and listen for DOM/Model events.  Views can be used to render a single model to render or a collection.  
+1.  **Routers**:  Used for Single Page Applications, routers let you add paths to the URL so that you can maintain history and allow users to jump to a specific state in your single-page-application.  A router always contains at least one route and a function to map that particular route to.  
 
-![Backbone JS Arcitechture from https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwiknZLWnrDQAhULrVQKHdKdDtsQjxwIAw&url=http%3A%2F%2Fwww.slideshare.net%2Fronreiter%2Fwriting-html5-web-apps-using-backbonejs-and-gae&psig=AFQjCNHvjo7n-Xt_eytdroPc3zdRUVQY7g&ust=1479487463101423](images/backbonejs-architecture.jpg)
+![Backbone JS Architecture from https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwiknZLWnrDQAhULrVQKHdKdDtsQjxwIAw&url=http%3A%2F%2Fwww.slideshare.net%2Fronreiter%2Fwriting-html5-web-apps-using-backbonejs-and-gae&psig=AFQjCNHvjo7n-Xt_eytdroPc3zdRUVQY7g&ust=1479487463101423](images/backbonejs-architecture.jpg)
```

Comment:
Lets change this to say "from http://www.slideshare.net/ronreiter/writing-html5-web-apps-using-backbonejs-and-gae" instead.